lib/liquid/parse_context.rb

```diff
@@ -0,0 +1,39 @@
+module Liquid
+  class ParseContext
```

Comment:
I'm tempted to make this delegate `method_missing` to the `@options` hash so that it maintains full API compatibility with old tags. But that is hacky and I'm not sure this will actually break any tags given that it does have `[]`.